v1.0/openapi3/schemas/Money.yaml

```diff
@@ -0,0 +1,15 @@
+title: Money
+type: object
+description: 'Data model for the complex type Money.'
+properties:
+  currency:
+    type: string
```

Comment:
shouldn't it here be a reference to Currency type instead of string?

Reply:
Thanks, well spotted. I went and did one more pass of the ref's to make sure they are correct now.